src/web/mod.rs

```diff
@@ -429,15 +430,23 @@ fn style_css_handler(_: &mut Request) -> IronResult<Response> {
     Ok(response)
 }
 
-fn menu_js_handler(_: &mut Request) -> IronResult<Response> {
-    let mut response = Response::with((status::Ok, MENU_JS));
+fn load_js(file_path_str: &'static str) -> IronResult<Response> {
+    let mut response = Response::with((status::Ok, file_path_str));
     let cache = vec![CacheDirective::Public,
                      CacheDirective::MaxAge(STATIC_FILE_CACHE_DURATION as u32)];
     response.headers.set(ContentType("application/javascript".parse().unwrap()));
     response.headers.set(CacheControl(cache));
     Ok(response)
 }
 
+fn menu_js_handler(_: &mut Request) -> IronResult<Response> {
+    load_js(MENU_JS)
+}
+
+fn index_js_handler(_: &mut Request) -> IronResult<Response> {
+    load_js(INDEX_JS)
+}
```

Comment:
Maybe it would be easier to pass a closure to `static_resource` than to have this boilerplate?